rajawali/src/main/java/org/rajawali3d/util/ObjectColorPicker.java

```diff
@@ -71,24 +71,30 @@ public void unregisterObject(Object3D object) {
 		if (mObjectLookup.contains(object)) {
 			mObjectLookup.remove(object);
 		}
+		object.setPickingColor(Object3D.UNPICKABLE);
 	}
 
 	public void getObjectAt(float x, float y) {
-		mRenderer.getCurrentScene().requestColorPickingTexture(new ColorPickerInfo(x, y, this));
+		mRenderer.getCurrentScene().requestObjectPicking(new ColorPickerInfo(x, y, this));
 	}
 
 	public RenderTarget getRenderTarget() {
 		return mRenderTarget;
 	}
 
+	@Deprecated
```

Comment:
Ditch this method completely. I just made a fairly substantial change to the API and this is a trivial adjustment to make.